zebra-chain/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 #![doc(html_favicon_url = "https://www.zfnd.org/images/zebra-favicon-128.png")]
 #![doc(html_logo_url = "https://www.zfnd.org/images/zebra-icon.png")]
 #![doc(html_root_url = "https://doc.zebra.zfnd.org/zebra_chain")]
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
Nit: let's just delete unused attributes

They already look like shell comments, so adding another comment makes it hard to work out if they're active.
```suggestion
```